build.gradle

```diff
@@ -148,6 +148,8 @@ dependencies {
         exclude module: 'kryo' // use Spark's version
     }
 
+    compile 'org.broadinstitute:jbwa:bd72b489f9697a857671533c86ad5906967f8672-20160520.183811-1'
```

Comment:
Changed to `com.github.lindenb`, and uploaded the same snapshot with the updated groupID.